python/cugraph/cugraph/tests/components/test_connectivity.py

```diff
@@ -28,14 +29,17 @@
 
 import cudf
 import cugraph
-from cugraph.testing import utils, DATASETS, STRONGDATASETS
+from cugraph.testing import utils, DATASETS_TESTING
+from cugraph.datasets import dolphins, netscience, email_Eu_core
+
+
+DATASETS_BATCH = [dolphins, netscience, email_Eu_core]
 
 # Temporarily suppress warnings till networkX fixes deprecation warnings
 # (Using or importing the ABCs from 'collections' instead of from
 # 'collections.abc' is deprecated, and in 3.8 it will stop working) for
 # python 3.7.  Also, this import networkx needs to be relocated in the
 # third-party group once this gets fixed.
-import warnings
 
 with warnings.catch_warnings():
     warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
I think we should get rid of this now.

Reply:
Issue has been raised to address this concern. Since this isn't within the scope of the promotion of the `datasets` API, it will be resolved in a separate PR.